// [[Reason: Invalid computed property can only be detected by TSC]] ////

//// [tests/cases/conformance/async/es6/functionDeclarations/asyncFunctionDeclaration8_es6.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -4,17 +4,17 @@
 declare var v: invalid;
 
 /// [Errors] ////
 
-asyncFunctionDeclaration8_es6.ts(1,11): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncFunctionDeclaration8_es6.ts(1,12): error TS2304: Cannot find name 'await'.
 asyncFunctionDeclaration8_es6.ts(1,20): error TS2304: Cannot find name 'foo'.
+asyncFunctionDeclaration8_es6.ts(1,20): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 
 
 ==== asyncFunctionDeclaration8_es6.ts (3 errors) ====
     var v = { [await]: foo }
-              ~~~~~~~
-!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                ~~~~~
 !!! error TS2304: Cannot find name 'await'.
                        ~~~
\ No newline at end of file
-!!! error TS2304: Cannot find name 'foo'.
+!!! error TS2304: Cannot find name 'foo'.
+                       ~~~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
\ No newline at end of file
